runtime/caml/misc.h

```diff
@@ -215,13 +215,22 @@ Caml_inline void call_timing_hook(_Atomic caml_timing_hook * a)
 
 #endif /* CAML_INTERNALS */
 
+#if (defined (__STDC_VERSION__) && __STDC_VERSION__ >= 202311L) \
```

Comment:
For all these constructs, you should make sure they will be have as intended if the symbol is defined, but without a value (e.g. simply `#define __STDC_VERSION`). The usual idiom is to use instead of `#if defined(FOO) && FOO >= VALUE` is `#if defined(FOO) && (FOO - VALUE >= 0)`.

Reply:
I agree with the general case, but we're not using the idiom you mention on the rest of the compiler. A quick search on GitHub shows that very few projects actually check whether `__STDC_VERSION__` isn't a value. Some do `#if defined(FOO) && (FOO - 0 >= VALUE)`. Considering that it's well defined in the standard, that might be a bit over cautious?